### ISSUE TITLE
W0511 false positive

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#149]
Originally reported by: **Anonymous**

---

I'm using PyLint in Spyder2.2.1 and it throws a "W0511 warning" in following case, where it should stay silent:

```
#!python

    assert checkio(2888) == 'MMDCCCLXXXVIII'
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/149
